packages/icon/package.py

```diff
@@ -562,10 +566,8 @@ def configure_args(self):
                 cuda_host_compiler = self.compiler.cxx
                 cuda_host_compiler_stdcxx_libs = self.compiler.stdcxx_libs
 
-            config_vars['NVCFLAGS'].extend([
-                '-ccbin {0}'.format(cuda_host_compiler), '-g', '-O3',
-                '-arch=sm_{0}'.format(gpu)
-            ])
+            config_vars['NVCFLAGS'].extend(
```

Comment:
This breaks the variant `+cuda-gcc` since it relies on cuda_host_compiler.
Please find a way to not break it.